components/script/dom/node.rs

```diff
@@ -105,6 +106,9 @@ pub struct Node {
     /// A bitfield of flags for node items.
     flags: Cell<NodeFlags>,
 
+    /// The maximum version of any inclusive descendent of this node.
+    inclusive_descendents_version: Cell<u64>,
```

Comment:
Is this deliberately "descendents" instead of "descendants", here and elsewhere? I believe the latter is much more standard.